src/components/PostPageDetail/index.js

```diff
@@ -582,6 +586,11 @@ const PostPageDetailIdComponent = (props) => {
             isBackButton={true}
             source={SOURCE_PDP}
             height={getHeightHeader()}
+            isFollow={item?.is_following_target || parentData?.is_following_target}
+            onPressFollUnFoll={() => {
+              followUnfollow(item);
+              setItem({...item, is_following_target: !item?.is_following_target});
+            }}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isFollow` prop and `onPressFollUnFoll` handler are added to the component. Make sure that the `followUnfollow` function correctly updates the state and that the state update with `setItem` does not cause unnecessary re-renders.
```diff
+             isFollow={item?.is_following_target || parentData?.is_following_target}
+             onPressFollUnFoll={() => {
+               followUnfollow(item);
+               setItem({...item, is_following_target: !item?.is_following_target});
+             }}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->